guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+
+#### Headings
+
+We use sentence case so only the first letter is upper case. We never link headings.
+
+#### Quotation marks
+
+We use double quotation marks for:
+* exact quotations
+* direct speech.
+
+We use single quotation marks for:
+* technical terms (the first time it is used)
+* classification descriptors
+* titles of documents or publications
+
+
+## Links
+
+We:
+* link to relevant content on our website before we link to external websites 
+* never link headings
+* write descriptive links that tell you what you’ll find when you follow them - not 'click here'
+* don't set links to open in a new tab or window, with exceptions to be decided case by case.
+
+
+## Lists
+
+We use bulleted lists (coded as unordered lists) to list items or points, and numbered lists (coded as ordered lists) for processes where the order of steps is important.
+
+We try to:
+* keep our lists short (2–7 items)
+* only use 1 level of nesting.
+
+We use 2 types of bulleted lists – single-sentence lists and multi-sentence lists.
+
+When we’re writing a single-sentence list, we:
+* start with a stem sentence that all the points have in common
+* start each point in lower case, and only use a full stop on the last point
+* don't use 'and' or 'or' on the second-to-last point
+* check that each point makes a full sentence when read with the stem.
+
+Multi-sentence lists are introduced by a complete sentence.
+* Each point in the list is also a complete sentence.
+* Each point can be 1–3 sentences long.
+* Each point begins with a capital letter and ends with a full stop.
+
+
+## Numbers
+
+In general:
+* we use numerals instead of words when we write numbers – this helps you scan our content
+* we use commas and no spaces to separate thousands when the number is over 10,000
+* when we’re talking about numbers in the millions, we use the word 'million' instead of writing out the number in full
+* we use spaces to separate groups of numbers when we write phone numbers.
+
+
+## Symbols, currency and abbreviations
+
+#### Symbols
+
+We use:
+* % – not 'percent' or 'per cent'
+* & – only if it’s part of a brand name
+* KB for kilobyte, MB for megabyte, and GB for gigabyte, for example 122 KB.
+
+#### Currency
+
+We put both the currency code and currency symbol before any amounts of money we write.
+We don't use spaces between the code, symbol and amount.
+
+For example:
+* 'If you’re a United States citizen, you pay USD$640'.
+* 'If you’re an Australian citizen it costs AUD$890'.
+
+#### Abbreviations
+
+We expand all abbreviations when we use them for the first time on a page.
+For example: 'Welcome to MusicBrainz (MB)'.
+
+To make our content easier to read, we avoid using:
+* e.g.
+* i.e.
+* etc.
+
+These are often written in full, such as:
+* for example
+* such as
+* that is
+* and so on.
+
+
+## Common terms
+
+We refer to people who use our data, sites and applications as 'user'/'users'
```

Comment:
Would you expect to make a difference depending on what the document is about? Say, "voters" in documentation about voting or "editors" about editing? Or do you feel always "users" is better? I'm not 100% sure what I feel myself.

Reply:
Hmm MB is pretty different to a closed org that serves 'other people'. We have a bunch of overlap, so maybe we do need more.

"We use two terms to refer to the people who interact with our data, sites and applications:
User/s: Everyone, including the below
Editor/s: Users who contribute or are part of the community"

Maybe that's it? Not too complicated? I don't know if we need to list every option, like 'voters', or 'forum members' etc.

I think this line I was interested in adding because I've been in discussions at other orgs where it was 'user' vs 'patron' vs 'customer' vs 'client' vs 'consumer'. Most of those aren't applicable to MB anyway. But sometimes people say "but 'user' has negative connotations" and then it's nice to point to a sign (or a guideline).

btw this common terms section I would like to regularly expand as questions come up.

Reply:
Yeah, I think it makes sense to stick to something, but we're currently super inconsistent. See https://musicbrainz.org/user/aerozol (URL: "user/", title: "Editor") :)

Reply:
Ahhh! To be fair 'user' is a pretty universal term so it makes sense for it to be in the menu bar, and then 'editor' for more specific MB things. I think my edited version above could still work, as long as nobody uses this guide as a "this is the law" thing. I might add a note to the top to remind people that it's a guide.